protos/proto/aptos/indexer/v1/grpc.proto

```diff
@@ -0,0 +1,108 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package aptos.indexer.v1;
+
+import "aptos/indexer/v1/raw_data.proto";
+import "aptos/transaction/v1/transaction.proto";
+import "aptos/util/timestamp/timestamp.proto";
+
+message StreamProgressSampleProto {
```

Comment:
i feel Proto here is a little redundant; either remove or use `unit` something?

Reply:
I use here because in the rust code I have a corresponding struct called `StreamProgressSample`. Just want to have a different name here.